flw/src/main/java/org/motechproject/nms/flw/web/FrontLineWorkerImportController.java

```diff
@@ -63,6 +64,12 @@ public void updateFrontLineWorkersLanguage(@RequestParam MultipartFile csvFile)
     @RequestMapping(value = "/update/msisdn", method = RequestMethod.POST)
     @ResponseStatus(HttpStatus.OK)
     public void updateFrontLineWorkersMSISDN(@RequestParam MultipartFile csvFile) {
+
+        // Update FrontLineWorkers by Msisdn import from csv is not supported
+        if(true) {
```

Comment:
No need for the if(true) here. Just throw the exception and remove the rest of the code.